tock/tock/templates/hours/timecard_form.html

```diff
@@ -28,7 +28,13 @@
            {% if i.task_name %}
             assigned to {{ i.task_name }}
            {% endif %}
-        </li>
+         </li>
+        {% empty %}
+         <li>We're sorry, there was an error retrieving your data from Float.
```

Comment:
We could, if we wanted, surface the error here. I can see where that might not be desirable, though. Just a thought.

Reply:
Displaying the error here would be useless as this is user facing; folks aren't going to know what it means.  It's on us as the devs/admins to look in the logs and debug from there. :-)

Reply:
Correction:  displaying the raw error here.